consensus/src/consensus.rs

```diff
@@ -137,8 +198,10 @@ where
     ) -> JoinHandle<StoreResult<()>> {
         tokio::spawn(async move {
             let consensus_index = store.read_last_consensus_index()?;
+            let last_committed = store.read_last_committed();
```

Comment:
I would rename `last_committed` to something that makes it explicit this comes from a store, e.g. `recovered_last_committed`, here, and in all the places this is used. 

The soundness of your reconstructed `ConsensusState` only works if this table has been reconstructed from disk, so I think it's wise to use the name to invite the reader of `construct_dag_from_cert_store` to go read the very different place where this value is computed.